### PR TITLE
Add support for Lunar II

### PR DESCRIPTION
**Pull request by ai03-2725**
## Description

Adds support for the Lunar II keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Add support for Lunar II

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/lunar_ii/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Lunar II
+
+![lunar_ii](https://i.imgur.com/W3xbm4m.png)
```
Comment (drashna) [inline keyboards/ai03/lunar_ii/readme.md:3]:
```suggestion
![lunar_ii](https://i.imgur.com/W3xbm4mh.png)
```

File: keyboards/ai03/lunar_ii/info.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
+{
+    "keyboard_name": "Lunar II",
+    "manufacturer": "ai03 Design Studio",
+    "url": "https://ai03.com/projects/lunar-ii",
+    "maintainer": "ai03-2725",
+    "usb": {
+        "vid": "0xA103",
+        "pid": "0x0016",
+        "device_version": "1.0.0",
+        "polling_interval": 1
```
Comment (drashna) [inline keyboards/ai03/lunar_ii/info.json:10]: This is the default.

```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ai03/lunar_ii/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/ai03/lunar_ii/rules.mk:15]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

File: keyboards/ai03/lunar_ii/rules.mk [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/ai03/lunar_ii/rules.mk:17]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**